servers/physics_server.h

```diff
@@ -658,6 +658,43 @@ class PhysicsServer : public Object {
 	~PhysicsServer();
 };
 
+class PhysicsServerManager {
+	static Vector<String> physics_servers_classes;
+	static int default_class_id;
+	static int default_class_priority;
+
+public:
+	static const String setting_property_name;
+
+private:
+	static void on_classes_changed();
+
+public:
+	template <class T>
+	static void register_class() {
+
+		ERR_FAIL_COND(-1 != find_class_id(T::get_class_static()));
```

Comment:
It's a matter of preference, but we use `(stuff != -1)` everywhere else in Godot, so I'd prefer if you used it here too.

Reply:
Ok Thanks for advice